README.md

```diff
@@ -16,12 +16,79 @@ goose
+**Welcome to `goose`, the Open-Source, Extensible AI Coding Assistant!** 
 
-To solve problems, `goose` breaks down instructions into sequences of tasks and carries them out using tools. Its ability to connect its changes with real outcomes (e.g. errors) and course correct is its most powerful and exciting feature. `goose` is free open source software and is built to be extensible and customizable.
+Unlike traditional coding tools, `goose` leverages advanced natural language processing to interpret complex commands into actionable, executable sequences. This allows for a coding assistant that not only writes code but also understands the intent behind your commands, enabling it to manage broader project scopes autonomously.
+
+```
+TODO: quick install instructions will go here in a callout
+```
+
+`TODO: slow video down and center it on the page`
 
 ![goose_demo](https://github.com/user-attachments/assets/0794eaba-97ab-40ef-af64-6fc7f68eb8e2)
 
+## What Sets Goose Apart?
+
+* **Adaptability**: `goose` excels in its ability to adapt to changes in code in real-time, making necessary adjustments as errors occur. This mimics a human developer's approach to coding, iteratively refining the work until perfection is achieved.
+* **Autonomy**: While tools like Copilot provide line-by-line code suggestions based on comments or partial code snippets, goose can take full command descriptions to manage entire projects with little to no ongoing input from developers.
+* **Customization and Extensibility**: Open-source and fully customizable, goose is built to support extensive user-defined modifications, unlike the mostly closed systems of Copilot and Cursor. Developers can tailor `goose` extensively to fit specific workflows, integrate with various toolkits, and enhance functionality through community contributions.
+* **Privacy and Security**: Operating locally on your device, `goose` ensures that your proprietary code and sensitive data are never sent to the cloud.
+
+## Features and capabilities
+`#TODO: include table (?) but add some more descriptions on goose's capabilities`
+
+|  | goose | aider | copilot | cursor |
+| --- | --- | --- | --- | --- |
+| open source | ✅ | ✅ | ❌ | ❌ |
+| in terminal | ✅ | ✅ | ❌ | ❌ |
+| VSCode extension | ✅ | ✅ | ✅ | ❌ |
+| Jetbrains extension  | WIP | ✅ | ✅ | ❌ |
+| Anthropic | ✅ | ✅ | ❌ | ✅ |
+| OpenAI | ✅ | ✅ | ✅ | ✅ |
+| Ollama | ✅ | ✅ | ❌ | ❌ |
+| Gemini | ❌ | ✅ | ❌ | ❌ |
+| Bedrock | ✅ | ✅ | ❌ | ❌ |
+| Azure | ✅ | ✅ | ❌ | ❌ |
+| Cohere | ❌ | ✅ | ❌ | ❌ |
+| DeepSeek | ❌ | ✅ | ❌ | ❌ |
+| Groq | ❌ | ✅ | ❌ | ❌ |
+| VertexAI | ❌ | ✅ | ❌ | ❌ |
+| Databricks | ✅ | ❌ | ❌ | ❌ |
+| Custom response instructions | ✅ | ✅ | ❔ | ✅ |
+|  |  |  |  |  |
+
+### Examples
+
+`#TODO: rewrite/reformat`
+
+Here are some examples that have been used: 
+
+```
+G❯ Looking at the in progress changes in this repo, help me finish off the feature. CONTRIBUTING.md shows how to run the tests.
```

Comment:
Do we want to try to share the session for these or have gif's of this? 

Another option here is we could also point to the Discussion section seed it with some full examples/encourage others to share?

Reply:
I am not sure -- it's something i have thought a bit about but don't have a good idea on what to do. I looked at Aider and what they have is links to videos but I don't want people to have to click away from the readme to see the magic of goose ideally

Looking for opinions here definitely